sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluate/_utils.py

```diff
@@ -37,12 +37,6 @@ class AzureMLWorkspace(NamedTuple):
     workspace_name: str
 
 
-class EvaluateResult(TypedDict):
-    metrics: Dict[str, float]
```

Comment:
I'm not sure if we'll have non-float types for aggregated metrics in the future. Should we use Any for safety?

Reply:
![image](https://github.com/user-attachments/assets/f7c83e1a-dd50-4b66-a0ea-e0770b399b3f)
yup, that's already taken care of in the `EvaluationResult` that is public